main-app/app/scripts/MainController.js

```diff
@@ -1,22 +1,29 @@
-var app = angular.module("MainApp", [] );
+var app = angular.module("MainApp");
 
-app.controller("MainController", function($http, $scope) {
+app.controller("MainController", function($http, $scope, $location) {
```

Comment:
Not going to hold up the current PR for this (just the one issue above) - but we need to start splittng up into functional units as per yesterday's session.